Jenkinsfile

```diff
@@ -52,12 +52,12 @@ def init_git_win() {
 def make(docker_type, make_flag) {
   timeout(time: max_time, unit: 'MINUTES') {
     try {
-      sh "${docker_run} ${docker_type} make ${make_flag}"
+      sh "${docker_run} ${docker_type} --dockerbinary docker make ${make_flag}"
```

Comment:
what's --dockerbinary? why not put it into docker_run?

Reply:
Dockerbinary specifies whether the command has to be executed in nvidia-docker (bringing GPU support) or in regular docker (to run on CPU-only instances). We can't modify docker_run because it's a shared template and every task has to specify the docker-binary it has to run in. In the current CI, the docker-binary is determined automatically, depending on the name of the Dockerfile - but I'd like to follow a declarative approach instead of automatic assumptions and hardcoded names.